### PR TITLE
fix: tree dnd

### PR DESCRIPTION
**Pull request by Kupov**
https://jira.railsc.ru/browse/GOODS-503

### CODE REVIEW

File: dist/assets-help.bundle.js [hunk 1]
```diff
@@ -13,537 +13,537 @@
...
+/******/ 		var me = installedModules[moduleId];
+/******/ 		if(!me) return __webpack_require__;
+/******/ 		var fn = function(request) {
+/******/ 			if(me.hot.active) {
+/******/ 				if(installedModules[request]) {
+/******/ 					if(installedModules[request].parents.indexOf(moduleId) < 0)
+/******/ 						installedModules[request].parents.push(moduleId);
+/******/ 					if(me.children.indexOf(request) < 0)
+/******/ 						me.children.push(request);
+/******/ 				} else hotCurrentParents = [moduleId];
```
Comment (apress-hound) [inline dist/assets-help.bundle.js]: Expected '{' and instead saw 'hotCurrentParents'.<br>Too many errors. (1% scanned).

File: dist/assets-help.bundle.js [hunk 2]
```diff
@@ -13,537 +13,537 @@
...
+/******/ 	function hotCreateRequire(moduleId) { // eslint-disable-line no-unused-vars
+/******/ 		var me = installedModules[moduleId];
+/******/ 		if(!me) return __webpack_require__;
+/******/ 		var fn = function(request) {
+/******/ 			if(me.hot.active) {
+/******/ 				if(installedModules[request]) {
+/******/ 					if(installedModules[request].parents.indexOf(moduleId) < 0)
+/******/ 						installedModules[request].parents.push(moduleId);
+/******/ 					if(me.children.indexOf(request) < 0)
+/******/ 						me.children.push(request);
```
Comment (apress-hound) [inline dist/assets-help.bundle.js]: Expected '{' and instead saw 'me'.

File: dist/assets-help.bundle.js [hunk 3]
```diff
@@ -13,537 +13,537 @@
...
+/******/ 	var hotCurrentParents = []; // eslint-disable-line no-unused-vars
+/******/ 	
+/******/ 	function hotCreateRequire(moduleId) { // eslint-disable-line no-unused-vars
+/******/ 		var me = installedModules[moduleId];
+/******/ 		if(!me) return __webpack_require__;
+/******/ 		var fn = function(request) {
+/******/ 			if(me.hot.active) {
+/******/ 				if(installedModules[request]) {
+/******/ 					if(installedModules[request].parents.indexOf(moduleId) < 0)
+/******/ 						installedModules[request].parents.push(moduleId);
```
Comment (apress-hound) [inline dist/assets-help.bundle.js]: Expected '{' and instead saw 'installedModules'.

File: dist/assets-help.bundle.js [hunk 4]
```diff
@@ -13,537 +13,537 @@
...
+/******/ 	}
+/******/ 	
+/******/ 	var hotApplyOnUpdate = true;
+/******/ 	var hotCurrentHash = "8c3811797a8606c6e6a0"; // eslint-disable-line no-unused-vars
+/******/ 	var hotCurrentModuleData = {};
+/******/ 	var hotCurrentParents = []; // eslint-disable-line no-unused-vars
+/******/ 	
+/******/ 	function hotCreateRequire(moduleId) { // eslint-disable-line no-unused-vars
+/******/ 		var me = installedModules[moduleId];
+/******/ 		if(!me) return __webpack_require__;
```
Comment (apress-hound) [inline dist/assets-help.bundle.js]: Expected '{' and instead saw 'return'.<br>Identifier '__webpack_require__' is not in camel case.

File: dist/assets-help.bundle.js [hunk 5]
```diff
@@ -13,537 +13,537 @@
...
+/******/ 		Object.defineProperty({}, "x", {
+/******/ 			get: function() {}
+/******/ 		});
+/******/ 		canDefineProperty = true;
+/******/ 	} catch(x) {
+/******/ 		// IE will fail on defineProperty
+/******/ 	}
+/******/ 	
+/******/ 	var hotApplyOnUpdate = true;
+/******/ 	var hotCurrentHash = "8c3811797a8606c6e6a0"; // eslint-disable-line no-unused-vars
```
Comment (apress-hound) [inline dist/assets-help.bundle.js]: 'hotCurrentHash' was used before it was defined.<br>Mixed double and single quotes.

File: dist/assets-help.bundle.js [hunk 6]
```diff
@@ -13,537 +13,537 @@
...
+/******/ 			}
+/******/ 		};
+/******/ 	}
+
+/******/ 	
+/******/ 	
+/******/ 	// Copied from https://github.com/facebook/react/blob/bef45b0/src/shared/utils/canDefineProperty.js
+/******/ 	var canDefineProperty = false;
+/******/ 	try {
+/******/ 		Object.defineProperty({}, "x", {
```
Comment (apress-hound) [inline dist/assets-help.bundle.js]: Mixed double and single quotes.

File: dist/assets-help.bundle.js [hunk 7]
```diff
@@ -13,537 +13,537 @@
...
+/******/ 			if(request.readyState !== 4) return;
+/******/ 			if(request.status === 0) {
+/******/ 				// timeout
+/******/ 				callback(new Error("Manifest request to " + requestPath + " timed out."));
+/******/ 			} else if(request.status === 404) {
+/******/ 				// no update available
+/******/ 				callback();
+/******/ 			} else if(request.status !== 200 && request.status !== 304) {
+/******/ 				// other failure
+/******/ 				callback(new Error("Manifest request to " + requestPath + " failed."));
```
Comment (apress-hound) [inline dist/assets-help.bundle.js]: Mixed double and single quotes.

File: dist/assets-help.bundle.js [hunk 8]
```diff
@@ -13,537 +13,537 @@
...
+/******/ 			request.timeout = 10000;
+/******/ 			request.send(null);
+/******/ 		} catch(err) {
+/******/ 			return callback(err);
+/******/ 		}
+/******/ 		request.onreadystatechange = function() {
+/******/ 			if(request.readyState !== 4) return;
+/******/ 			if(request.status === 0) {
+/******/ 				// timeout
+/******/ 				callback(new Error("Manifest request to " + requestPath + " timed out."));
```
Comment (apress-hound) [inline dist/assets-help.bundle.js]: Mixed double and single quotes.

File: dist/assets-help.bundle.js [hunk 9]
```diff
@@ -13,537 +13,537 @@
...
+/******/ 			var request = new XMLHttpRequest();
+/******/ 			var requestPath = __webpack_require__.p + "" + hotCurrentHash + ".hot-update.json";
+/******/ 			request.open("GET", requestPath, true);
+/******/ 			request.timeout = 10000;
+/******/ 			request.send(null);
+/******/ 		} catch(err) {
+/******/ 			return callback(err);
+/******/ 		}
+/******/ 		request.onreadystatechange = function() {
+/******/ 			if(request.readyState !== 4) return;
```
Comment (apress-hound) [inline dist/assets-help.bundle.js]: Expected '{' and instead saw 'return'.

File: dist/assets-help.bundle.js [hunk 10]
```diff
@@ -13,537 +13,537 @@
...
+/******/ 		head.appendChild(script);
+/******/ 	}
+/******/ 	
+/******/ 	function hotDownloadManifest(callback) { // eslint-disable-line no-unused-vars
+/******/ 		if(typeof XMLHttpRequest === "undefined")
+/******/ 			return callback(new Error("No browser support"));
+/******/ 		try {
+/******/ 			var request = new XMLHttpRequest();
+/******/ 			var requestPath = __webpack_require__.p + "" + hotCurrentHash + ".hot-update.json";
+/******/ 			request.open("GET", requestPath, true);
```
Comment (apress-hound) [inline dist/assets-help.bundle.js]: Mixed double and single quotes.

**Comment by Kupov** [comment]
@dontuse

File: dist/assets-help.bundle.js [hunk 11]
```diff
@@ -78,7 +78,7 @@ return /******/ (function(modules) { // webpackBootstrap
 /******/ 	}
 /******/ 	
 /******/ 	var hotApplyOnUpdate = true;
-/******/ 	var hotCurrentHash = "e2a523789b25597b543a"; // eslint-disable-line no-unused-vars
+/******/ 	var hotCurrentHash = "6c1fed40a9eebc91f6d6"; // eslint-disable-line no-unused-vars
```
Comment (apress-hound) [inline dist/assets-help.bundle.js]: 'hotCurrentHash' was used before it was defined.<br>Mixed double and single quotes.

**Comment by Kupov** [comment]
не принимай пока еще багу нашол

**Comment by dontuse** [comment]
ok

**Comment by cynek** [comment]
не забывайте ребейзы

**Pull request merged**